client/src/components/QA/QA.jsx

```diff
@@ -56,6 +57,7 @@ class QA extends React.Component {
   }
 
   updateData() {
+    console.log(`❓❓❓GET request for Questions-product ID: ${this.props.selectedProduct.id}❓❓❓`)
```

Comment:
Why not only call this in one method, and call that method from `fetchData` & `updateData` methods? If you do that, then any changes such as this only need to occur in one place, which is easier to update and less error-prone in updates.